architecture-decision-record/0027-use-member-cicd-access-for-configuration-management.md

```diff
@@ -0,0 +1,41 @@
+# 27. Use Member CICD Access for Configuration Management
+
+Date: 2023-08-17
+
+## Status
+
+✅ Accepted
+
+## Context
+
+Users of the Modernisation Platform often host applications which require additional configuration on top of standard infrastructure provisioning.
+
+This configuration whilst sometimes manual often can be achieved via code, eg scripts or running a configuration management tool such as Ansible on the server.
+
+## Options
+
+#### Run configuration management tools manually on servers
+
+This works but is not best practice and runs the risk of things not being applied consistently.
+
+#### Run configuration management from the environments repository pipeline
+
+Automated solution and keeps everything in one place, but begins to mix infrastructure with configuration which we do not want to do. The environments repository is already very large and this will add to this.
+
```

Comment:
i am a little lost in what this paragraph is trying to say

is it something on the lines of configuration management  will add this?

Reply:
So it's kind of a pros and cons statement, but in a paragraph, so I could have also written like:

**Pros**
 - Automated
 - Everything in one place
 
**Cons**
 - Mixes infrastructure and configuration management
 - Adds to already large repository

I was trying to keep it short but happy to do either way?